src/basic/cgroup-util.c

```diff
@@ -2211,6 +2212,62 @@ int cg_trim_everywhere(CGroupMask supported, const char *path, bool delete_root)
         return 0;
 }
 
+int cg_mask_to_string(CGroupMask mask, char **ret) {
+        _cleanup_strv_free_ char **t = NULL;
+        CGroupController c;
+        char *s;
+
+        assert(ret);
+
+        if (!mask)
```

Comment:
we generally follow the rule that return parameters have to be initialized on success, and aren't touched on failure. In this case you forgot to reset *ret to NULL, on success.